ush/forecast_postdet.sh

```diff
@@ -678,12 +678,21 @@ MOM6_postdet() {
   echo "SUB ${FUNCNAME[0]}: MOM6 after run type determination"
 
   # Copy MOM6 ICs
-  ${NLN} "${COM_OCEAN_RESTART_PREV}/${PDY}.${cyc}0000.MOM.res.nc" "${DATA}/INPUT/MOM.res.nc"
+  if [[ "${DOIAU}" = "YES" ]]; then
+    sCDATE=$(date -d "${current_cycle:0:8} ${current_cycle:8:2} - 3 hours" +%Y%m%d%H)
+    sPDY="${sCDATE:0:8}"
+    scyc="${sCDATE:8:2}"
+  else
+    sPDY=${PDY}
+    scyc=${cyc} 
+  fi
```

Comment:
Are we sure this is needed?  `ush/forecast_predet.sh` sets these variables in the function `FV3_predet` [lines 219-231](https://github.com/guillaumevernieres/global-workflow/blob/f5daa23aefa238d8447b9c6437515593bc366700/ush/forecast_predet.sh#L219)and they are not local; meaning they are already in the environment when this function is called.

Same comment to follow for `CICE_postdet` below.

Reply:
I moved the definition of `sCDATE`, `sPDY`, ... into `common_predet` as you suggested below @aerorahul . More precisely, I copy/pasted  [L219-231 of forecast_predet.sh](https://github.com/guillaumevernieres/global-workflow/blob/f5daa23aefa238d8447b9c6437515593bc366700/ush/forecast_predet.sh#L219) into `common_predet`.